benchmark/Gemfile

```diff
@@ -24,3 +24,5 @@ gem 'representable'
 gem 'simple_ams'
 gem 'sqlite3', '~> 1.4'
 gem 'turbostreamer'
+
+gem 'heap-profiler'
```

Comment:
Accidentally added?

Reply:
oh, yup, sorry, the next PR has been leaked